### PR TITLE
Remove the forge shell and update the bootstrap project to no longer depend on it

### PR DESCRIPTION
**Pull request by jsight**

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/181/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java' /><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/181/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport'>org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/181/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport/org.jboss.windup.qs.rules/EjbBeanUtilsAsyncUsageRuleProviderTest/testRules/'><strong>org.jboss.windup.qs.rules.EjbBeanUtilsAsyncUsageRuleProviderTest.testRules</strong></a></li></ul>
Test FAILed.

File: bootstrap/src/main/java/org/jboss/windup/bootstrap/Bootstrap.java [hunk 1]
```diff
@@ -163,7 +168,7 @@ else if ("--batchMode".equals(arg) || "-b".equals(arg))
             }
             else if ("--evaluate".equals(arg) || "-e".equals(arg))
             {
-                isEvaluate = true;
+                System.out.println("\"" + arg + "\" is no longer required!");
```
Comment (lincolnthree) [inline bootstrap/src/main/java/org/jboss/windup/bootstrap/Bootstrap.java]: Good idea. Might want to also print help/usage after this is displayed.

Comment (lincolnthree) [inline bootstrap/src/main/java/org/jboss/windup/bootstrap/Bootstrap.java]: If they have not provided required inputs.

Comment (jsight) [inline bootstrap/src/main/java/org/jboss/windup/bootstrap/Bootstrap.java]: I believe that it does. If there is no command and no windup options, it is supposed to print a help message. I think it even works. :)

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/814/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/815/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/182/<h2>Failed Tests: <span class='status-failure'>1</span></h2><h3><a name='org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java' /><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/182/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport'>org.jboss.windup.quickstarts:windup-ejb-beanutils-async-rules-java</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://10.3.8.5/job/windup-core-PR-dependents-check/182/org.jboss.windup.quickstarts$windup-ejb-beanutils-async-rules-java/testReport/org.jboss.windup.qs.rules/EjbBeanUtilsAsyncUsageRuleProviderTest/testRules/'><strong>org.jboss.windup.qs.rules.EjbBeanUtilsAsyncUsageRuleProviderTest.testRules</strong></a></li></ul>
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/824/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/825/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/192/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/193/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/826/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/194/
Test PASSed.

**Comment by jsight** [comment]
Related PR: https://github.com/windup/windup-distribution/pull/8

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/827/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/195/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/829/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/197/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/831/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/199/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/833/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/201/
Test PASSed.

**Comment by lincolnthree** [comment]
Merged

**Pull request closed without merging**